packages/web-components/src/appointment-planner/client/components/appointment-planner.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  import { weekStore } from '../core/store/week-store'
+  import Fa from 'svelte-fa'
+  import { faChevronRight, faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+  import * as commonStyles from '../../../common/styles/index'
+  import cx from 'classnames'
+
+  import DateTimePicker from './date-time-picker.svelte'
+
+  import { apointmentPlannerSvg } from '../styles/appointment-planner.styles'
+  import {injectGlobalCss} from '../core/theme'
```

Comment:
Don't use global CSS - there will be multiple widgets and this will conflict - also may mess with third party CSS